src/coreclr/vm/codeman.h

```diff
@@ -1167,6 +1177,22 @@ class ExecutionManager
     friend class ClrDataAccess;
 #endif
 
+    enum
+    {
+        RangeSectionHandleArrayInitialSize = 100,
+        RangeSectionHandleArrayExpansionFactor = 2
```

Comment:
In debug/checked builds, we should make the initial size small and make the array grow on element at a time to ensure that the array resize algorithm and the potential race conditions are exercised.